its/projects/AutoGeneratedFiles/AssemblyInfo.cs

```diff
@@ -1,35 +1,35 @@
-﻿// <auto-generated>

```

Comment:
It looks like the changes from java ITs were merged. Could you please revert them?